save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestExecutionService.kt

```diff
@@ -352,6 +356,10 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
         }
     }
 
+    private fun Long.increaseIfApplicable(delta: Long?) = takeUnless { CountWarnings.isNotApplicable(it.toInt()) }
```

Comment:
Nariman also love takeUnless, but I do not think it is more readable as straight-forward `if`

Reply:
Yeah, but diktat will turn a straightforward `if` into a messy `?.let { } ?: `